test/rest_api/test_repair_task.py

```diff
@@ -0,0 +1,68 @@
+import sys
+
+# Use the util.py library from ../cql-pytest:
+sys.path.insert(1, sys.path[0] + '/../cql-pytest')
+from util import new_test_table, new_test_keyspace
+from rest_util import set_tmp_task_ttl
+from task_manager_utils import list_tasks, drain_module_tasks, get_task_status, check_child_parent_relationship
+
+empty_id = "00000000-0000-0000-0000-000000000000"
+module_name = "repair"
+long_time = 1000000000
+
+def run_repair_and_wait(rest_api, keyspace):
+    resp = rest_api.send("POST", f"storage_service/repair_async/{keyspace}")
```

Comment:
How many nodes are started in the test?

Reply:
For rest api test only one node is started.

Here is a dtest draft run on more nodes: https://github.com/scylladb/scylla-dtest/pull/2992. I guess it should use more sessions/keyspaces/tables.

Reply:
> For rest api test only one node is started.
The test.py infrastructure supports starting more than one node. You could ask @kbr-scylla for help how to do that. Since this is a new test, I hope we could start using the new test infrastructure. Start two nodes so we can test a real repair in the test. 

> Here is a dtest draft run on more nodes: scylladb/scylla-dtest#2992. I guess it should use more sessions/keyspaces/tables.

Yes, at least more tables per keyspaces.